templates/artifactory-oss/0/docker-compose.yml

```diff
@@ -0,0 +1,17 @@
+version: '2'
+
+services:
+
+  artifactory:
+    image: docker.bintray.io/jfrog/artifactory-oss:5.1.3
+    ports:
+     - "8081"
```

Comment:
Thanks for the review :)
What if we don't want to map port but use a load balancer to serve Artifactory over HTTPS?